docs/source/index.rst

```diff
@@ -46,20 +16,308 @@ Documentation
+
+        - :ref:`The pipeline operator<pipeline>` (``pipeline``)
+
+        - Transformations
+            + :ref:`Helmert transform<helmert>` (``helmert``)
+            + Horner real and complex polynomial evaluation (``horner``)
+            + :ref:`Horizontal gridshift<hgridshift>` (``hgridshift``)
+            + :ref:`Vertical gridshift<vgridshift>` (``vgridshift``)
+            + :ref:`Molodensky transform<molodensky>` (``molodensky``)
+            + :ref:`Kinematic gridshift with deformation model<deformation>` (``deformation``)
+
+        - Conversions
+            + :ref:`Unit conversion<unitconvert>` (``unitconvert``)
+            + :ref:`Axis swap<axisswap>` (``axisswap``)
+
+        - Projections
+            + :ref:`Central Conic projection<ccon>` (``ccon``)
+
+    * Significant documentation updates, including
+
+        - Overhaul of the structure of the documentation
+        - A better introduction to the use of PROJ
+        - :ref:`A complete reference to the new API<reference>`
+        - a complete rewrite of the section on geodesic calculations
+        - Figures for all projections
+
+    * New "free format" option for operation definitions, which permits separating tokens by whitespace when specifying key/value- pairs, e.g. ``proj = merc lat_0 = 45``.
+
+    * Added metadata to init-files that can be read with the :c:func:`proj_init_info` function in the new ``proj.h`` API.
+
+    * Added ITRF2000, ITRF2008 and ITRF2014 init-files with ITRF transformation parameters, including plate motion model parameters.
+
+    * Added ellipsoid parameters for GSK2011, PZ90 and "danish". The latter is similar to the already supported andrae ellipsoid, but has a slightly different semimajor axis.
+
+    * Added Copenhagen prime meridian.
+
+    * Updated EPSG database to version 9.2.0.
+
+    * Geodesic library updated to version 1.49.2-c.
+
+    * Support for analytical partial derivatives has been removed.
+
+    * Improved performance in Winkel Tripel and Aitoff.
+
+    * Introduced ``pj_has_inverse()`` function to ``proj_api.h``. Checks if an operation has an inverse. Use this instead of checking whether ``P->inv`` exists, since that can no longer be relied on.
+
+    * ABI version number updated to 13:0:0.
+
+    * Removed support for Windows CE.
+
+    * Removed the VB6 COM interface.
+
+    Bug fixes
+    ***************************************************************************
+
+    * Fixed incorrect convergence calculation in Lambert Conformal Conic. (`#16 <https://github.com/OSGeo/proj.4/issues/16>`_)
+
+
+    * Handle ellipsoid parameters correctly when using ``+nadgrids=@null``. (`#22 <https://github.com/OSGeo/proj.4/issues/22>`_)
+
+
+    * Return correct latitude when using negative northings in Transverse Mercator. (`#138 <https://github.com/OSGeo/proj.4/issues/138>`_)
+
+
+    * Return correct result at origin in inverse Mod. Stererographic of Alaska. (`#161 <https://github.com/OSGeo/proj.4/issues/161>`_)
```

Comment:
Not sure if it's too late to correct "Stererographic" -> "Stereographic" here and the other two a few lines down..